packages/scss/src/components/inputs/textfield/_textfield.material.scss

```diff
@@ -0,0 +1,121 @@
+%input_material {
+	padding-top: _component("textfield.material.padding-top");
+
+	.textfield-input {
+		background: transparent;
+		border-radius: 0;
+		box-shadow: 0 1px 0 0 _component("textfield.border.color");
+		padding: {
+			left: 0;
+			right: 0;
+		}
+
+		&::placeholder {
+			opacity: 0;
+		}
+
+		&:focus {
+			&::placeholder {
+				opacity: 1;
+			}
+
+			~ .textfield-label {
+				font-size: _theme("sizes.small.font-size");
+				top: 0;
+			}
+		}
+	}
+
+	.textfield-label {
+		color: _component("textfield.label.color");
+		font-size: _component("textfield.font-size");
+		font-weight: 400;
+		left: 0;
+		pointer-events: none;
+		position: absolute;
+		top: 1.3rem;
+		width: 100%;
+	}
+
+	.textfield-suffix {
+		position: absolute;
+		right: 0;
+		line-height: 1;
+		top: _component("textfield.material.padding-top") + _component("textfield.input.padding-vertical");
+	}
+
+
+	// PALETTE
+	// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+	// Coloring
+
+	@mixin inputColoring($palette) {
+		.textfield-input:focus {
+			box-shadow: 0 2px 0 0 _get($palette, "color");
+
+			~ .textfield-label {
+				color: _get($palette, "color");
+			}
+		}
+
+		// Needed when textfield label is not next to the textfield-input
+		&.is-focused {
+			.textfield-label {
+				color: _get($palette, "color");
+			}
+		}
+	}
+
+	// default color
+	@include inputColoring(_component("textfield.material.default-palette"));
+
+	// palette colors
+	@each $name, $palette in _palettes() {
+		&.palette-#{$name} {
+			@include inputColoring($palette);
+		}
+	}
+
+
+	// MOD
+	// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+	&.mod-autocomplete, &.mod-select, &.mod-search {
+		&::after {
+			right: 0;
```

Comment:
should use the same value as textfield-suffix

Reply:
textfield material suffix is now defined as right: 0

Reply:
✔️